src/ai/backend/manager/defs.py

```diff
@@ -80,3 +81,7 @@ class LockID(enum.IntEnum):
 
 DEFAULT_KEYPAIR_RESOURCE_POLICY_NAME: Final = "default"
 DEFAULT_KEYPAIR_RATE_LIMIT: Final = 10000
+
+DEFAULT_SHARED_MEMORY_SIZE: Final = "64m"
+DEFAULT_MIN_MEM_SHARED_MEM_RATIO: Final = Decimal(1.0)
+MIN_MEM_SHARED_MEM_RATIO_KEY: Final = "manager/mem-shmem-ratio"
```

Comment:
Please update the module-level comment to describe the schema of this new etcd key in `src/ai/backend/manager/config.py`.

Also, the key and the variable should be renmaed to align with `resources/max-shm-ratio`. The ratio has the max limit, while allowing zero allocation of shm if intended. Conceptually, this is not a manager-component option but a resource-control option.

Reply:
- I renamed the constants to `MAX_...` and related variables were also renamed. 
- Changed the comparative formula that checks too-large-shmem from `if mem / shmem <= MIN_RATIO` to `if shmem >= mem * MAX_RATIO`
- The constants are relocated to `src/ai/backend/common.defs.py`.